aldryn_config.py

```diff
@@ -160,6 +160,7 @@ def domain_settings(self, data, settings, env):
         if not domains and domain:
             domains = {
                 1: {
+                    'name': env('ALDRYN_PROJECT_NAME', ''),
```

Comment:
something like ``SITE_NAME`` (consistent with ``DOMAIN``) or ``ALDRYN_SITES_NAME`` (consistent with the other ``ALDRYN_SITES_*`` env vars.